source/container.puml

```diff
@@ -22,6 +24,10 @@ Rel(geoasset_api, eventengine, "Listens in order to get last updated info", "Clo
 }
 
 Rel(geonorge, csapi, "Harvests metadata", "CSW")
+Rel(datanorge, api_met_no, "Harvest api.met.no metadata catalog", "HTTP (DCAT)")
```

Comment:
Shouldn't there also be a connection to the Dynamic Geo-Assets API here, for registration? @havardf 

Apart from that, maybe we can remove this diagram now? Shouldn't it anyway be exchanged with a combination of second and third iterations?

Reply:
Well, we agreed that registrering a catalog is such a rare event that it seems like overkille to have an api for it. A config file thats manually updated is enough. And manual updates doesnt really fit into this diagram I think.

I guess it would. Its anyway better to not have multiple similar diagrams, because its confusing who is authoritative.